fast64_internal/oot/room/properties.py

```diff
@@ -172,7 +172,8 @@ def draw_props(self, layout: UILayout, dropdownLabel: str, headerIndex: int, obj
                 if self.roomShape == "ROOM_SHAPE_TYPE_IMAGE":
                     self.drawBGImageList(general, objName)
                 if self.roomShape == "ROOM_SHAPE_TYPE_CULLABLE":
-                    general.label(text="Cull regions are generated automatically.", icon="INFO")
+                    general.label(text="This is for CPU culling, and requires meshes to be parented to Custom Cull Group empties.", icon="INFO")
```

Comment:
```suggestion
                    general.label(text="The 'Cullable' room shape type is for CPU culling, and requires meshes to be parented to Custom Cull Group empties.", icon="INFO")
```
otherwise it's unclear what "this" refers to imo